package/docker/scripts/docker_publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+IMPLEMENTER_INTERFACE_IMAGE_TAG=${BAHMNI_VERSION:?}-${GITHUB_RUN_NUMBER:?}
+echo ${DOCKER_HUB_TOKEN} | docker login -u ${DOCKER_HUB_USERNAME} --password-stdin
+
+echo "[INFO] Pushing build images"
+docker push bahmni/implementer-interface:${IMPLEMENTER_INTERFACE_IMAGE_TAG}
+
+echo "[INFO] Tagging build images as SNAPSHOT Images"
+IMPLEMENTER_INTERFACE_SNAPSHOT_IMAGE_TAG=${BAHMNI_VERSION}-SNAPSHOT
```

Comment:
why are we using SNAPSHOT tag with version? didnt we agreed to use `latest` e.g. `implementer-interface:latest`